packages/ods/src/style/_progress-bar.scss

```diff
@@ -0,0 +1,26 @@
+
+$ods-progress-bar-border-radius: 8px;
+
+@mixin ods-progress-bar() {
+  appearance: none;
+  border: none;
+  border-radius: $ods-progress-bar-border-radius;
+  background-color: var(--ods-color-neutral-100);
+  width: 320px;
```

Comment:
why this value?

Reply:
It's the value in the figma

Reply:
The value in Figma is only for rendering there, you can leave default native rendering here and remove this